docs/content/latest/deploy/multi-dc/2dc-deployment.md

```diff
@@ -24,126 +23,128 @@ showAsideToc: true
 
-* `hybrid_clock_time`: The hybrid clock timestamp on the producer's tablet-server.
-* `last_read_hybrid_time`: The hybrid clock timestamp of the latest transaction pulled from the producer.
-
-An example script [`determine_replication_lag.sh`](/files/determine_replication_lag.sh) calculates replication lag for you. 
-The script requires the [`jq`](https://stedolan.github.io/jq/) package.
-
-In the example below, a replication lag summary is generated for all tables on a cluster. You can also request an individual table:
-```bash
-$ ./determine_repl_latency.sh -h
-determine_repl_latency.sh -m MASTER_IP1:PORT,MASTER_IP2:PORT,MASTER_IP3:PORT [ -c PATH_TO_SSL_CERTIFICATE ] (-k KEYSPACE -t TABLENAME | -a) [ -p PORT ] [ -r report ] [ -o output ] [ -u units ]
-Options:
-  -m | --master_addresses       Comma separated list of master_server ip addresses with optional port numbers [default 7100]
-  -c | --certs_dir_name         Path to directory containing SSL certificates if TLS node-to-node encryption is enabled
-  -k | --keyspace               Name of keyspace that contains the table that is being queried. YSQL keyspaces must be prefixed with ysql
-  -t | --table_name             Name of table
-  -a | --all_tables             Specify this flag instead of -k and -t if you want all tables in the universe included
-  -p | --port                   Port number of tablet server metrics page [default 9000]
-  -r | --report_type            Possible values: all,detail,summary [default all]
-  -o | --output_type            Possible values: report,csv [default report]
-  -u | --units                  Possible values: us (microseconds), ms (milliseconds) [default ms]
-  -h | --help                   This message
+An example script [`determine_replication_lag.sh`](/files/determine_replication_lag.sh) calculates replication lag for you. The script requires the [`jq`](https://stedolan.github.io/jq/) package.
 
+The following example generates a replication lag summary for all tables on a cluster. You can also request an individual table.
 
+```sh
 ./determine_repl_latency.sh -m 10.150.255.114,10.150.255.115,10.150.255.113
 ```
 
+Call `determine_repl_latency.sh -h` to get a summary of all command options.
+
 {{< /tip >}}
+
+**If you're using Yugabyte Platform to manage universes**, continue to the next section. If not, you're done!
+
+## Enable universes in Platform
+
+If you're using Yugabyte Platform to manage your universes, you need to call the following REST API endpoint on your Platform instance _for each universe_:
+
+```http
+PUT /api/customers/<myUUID>/universes/<myUniUUID>/setup_universe_2dc
+```
+
+Where `myUUID` is your customer UUID, and `myUniUUID` is the UUID of the universe (producer or consumer). The request should include an `X-AUTH-YW-API-TOKEN` header with your Platform API key.
```

Comment:
It would be nice to have some documentation eventually on how to get the API key in the platform UI and then link those docs here IMO. But this should be fine for now since we'll usually have customer support/ATS running customers through the setup process for now + they'll know how to generate the API keys.

Reply:
Lets add a screenshot for generating an API token and user permission needed for calling the API

Reply:
Need steps on how to find the customer and universes UUID

Reply:
@chirag-yb my worry about including those instructions in our docs now is that it might open a pandora's box on using YW APIs before we're ready for them to be publicly consumed. Thoughts?